components/Footer.tsx

```diff
@@ -20,6 +21,14 @@ const getFooterStyles = (_theme: NucaCustomTheme) =>
       flex: 1,
       width: undefined,
     },
+    zoomedImage: {
+      paddingTop: 12,
+      height: 375,
+      minHeight: 430,
+      flex: 1,
+      width: undefined,
```

Comment:
why is the width `undefined` ?

Reply:
I didn't personally set the width to `undefined`. However, after a closer look, I found out that the reason why the width is set to `undefined` is to ensure that the bottom banner image takes up the width of its parent component i.e. the container. The [answer in this thread](https://w3schools.invisionzone.com/topic/57959-when-an-elements-width-is-undefined-what-determines-it/?do=findComment&comment=318035) explains the issue. 

I tried to see what happens if the `width: undefined` is set to `width: '100%'` and the result is that on some devices selected in the developer console in Chrome, the bottom banner is properly displayed, so the `undefined` setting was totally pointless (setting the width to `undefined` actually makes the bottom banner invisible on some devices, or at least that's what the developer console screen size settings indicate).

Anyway, good catch!

Reply:
I just pushed a new commit that covers the width issue that was mentioned.

Changes:

1. the 'width' prop is now set to '100%' instead of 'undefined'
2. the footer image/bottom banner is now positioned absolutely at a distance of 0 units from the bottom of the parent container when the screen is large and positioned at a distance of '-20%' from the bottom of the screen on small devices (this was tested on all devices in the Chrome developer console, from which only "Surface Duo" does not display the banner as expected)
3. the NucaFormButton has a zIndex of 2 and the footer image has a zIndex of -1 => on some devices in the Chrome console, the banner is displayed on top of the form contents
4. the ScrollView is not scrollable and its height is now set to '100%' instead of '1000%'

Reply:
The Surface Duo issue is now fixed.

https://github.com/crafting-software/nuca-mobile/assets/32801760/520b391a-10a3-4a27-b612-52fb63a15788

Reply:
The banner still looks weird on iPhone 4 (although to be fair, that's a very old device and it is [discontinued](https://en.wikipedia.org/wiki/IPhone_4) anyways)

Reply:
@sutgeorge just a mention: this is just the *web* preview of how the app looks like on those devices, given the viewport size.
The iOS app could look slightly different on those devices, but i highly doubt it's installable on them.

Reply:
It is definitely not installable, especially on iPhone 4, since it is discontinued. Also, other devices such as Blackberry phones are probably not even used in 2024 either, due to the same reason previously mentioned (discontinuation/deprecation).

Reply:
Still, it is worth trying out multiple devices for the sake of ensuring that the app is responsive on several screen sizes.